### ISSUE TITLE
Status

### ISSUE THREAD

**Issue by EisenbergEffect** [arv/ecmascript-object-observe#25]
Hello. How are we doing on this? What is need to move this forward past a stage 2 proposal? How can I (and the Aurelia team) help you out?

**Comment by arv** [comment]
Right now this is missing a champion at TC39. So basically it means that this is on hold until someone takes over.

**Comment by EisenbergEffect** [comment]
Where did the process leave off? What would be required of such a champion at this point in the process? Also, can that role be fulfilled by someone who isn't a "paying member"? I'm interested to help get this moving again in whatever way I can.

**Comment by bterlson** [comment]
I though @ajklein was championing this (at least until November)? Anyway, non-members can be champions afaik, but it would be very difficult to push the proposal through the process without a strong advocate on the TC39 committee.

**Comment by ajklein** [comment]
Indeed, I'm the TC39 member responsible for Object.observe. @EisenbergEffect, I'd be interested in hearing more about your use case(s) for the feature, though this issue tracker's not the ideal place. Feel free to drop a line at adamk@chromium.org

**Comment by EisenbergEffect** [comment]
Saw your comment that you intend to withdrawal the proposal. I've heard a lot of alarmed developers within the community as a result. I'd like to know if there's still an opportunity to pick this up and see it through....even if it means re-working the design.

**Comment by ajklein** [comment]
@EisenbergEffect I don't think there's anything about my withdrawing Object.observe that would prevent someone else from picking it up again (or another spec like it), though spec champions must be members of TC39. The process would be similar to creating a new proposal, for which the process is laid out in https://github.com/tc39/ecma262#contributing-new-proposals.

As for alarmed developers, I'd be interested in hearing more detail about that; as noted previously, feel free to contact me off-github if you have specific concerns.
